bluepymm/prepare_combos/main.py

```diff
@@ -124,3 +124,5 @@ def add_parser(action):
     parser.add_argument('conf_filename')
     parser.add_argument('--continu', action='store_true',
                         help='continue from previous run')
+    parser.add_argument('--n_processes', help='number of processes',
```

Comment:
small nit: in the future, can we use dashes instead of underscores? It's more 'standard', I think.

Reply:
Thanks, duly noted.